docs/kratos/self-service/flows/user-logout.mdx

```diff
@@ -3,27 +3,51 @@ id: user-logout
 title: User logout
 ---
 
-Ory Identities supports two logout flows:
+```mdx-code-block
+import { apiFlow, browserFlow, spaFlow } from "./code/logout"
+import CodeTabs from "@site/src/theme/Code/CodeTabs"
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+```
 
-- Browser-based (easy): This flow works for all applications running on top of a browser. Websites, single-page apps,
-  Cordova/Ionic, and so on.
-- API-based (advanced): This flow works for native applications like iOS (Swift), Android (Java), Microsoft (.NET), React Native,
-  Electron, and others.
+The logout flow is used to terminate an Ory Identities session.
 
-import { apiFlow, browserFlow } from "./code/logout"
-import CodeTabs from "@site/src/theme/Code/CodeTabs"
+Ory supports logout on all supported clients:
+
+- [Server rendered browser applications](#server-side-browser-applications)
+- [Client side browser applications](#client-side-browser-applications)
+- [Native applications](#native-api-applications)
 
-## Self-service logout for server-side browser applications
+## Server-side browser applications
 
-To ensure that a logout was indeed intended by the user, Ory Identities first generates a Logout URL for a given Ory Session
-Cookie. You can then open the Logout URL in the Browser.
+To ensure that a logout was intended by the user, Ory Identities first generates a logout URL for a given Ory Session cookie. You
+can then open the logout URL in the browser.
 
-import SelfServiceLogoutBrowserFlow from "@site/src/theme/SelfServiceLogoutBrowserFlow"
+In practice, it is recommended to call the endpoint every time you want to show a logout button to the user.
 
-<SelfServiceLogoutBrowserFlow />
+Alternatively, you can create the logout URL only when the user actually clicks the logout button and immediately call the
+resulting `logout_url` from your server to log out the user. Beware, that this method can lead to inconsistent behavior for users
+that switch accounts often.
+
+<CodeTabs items={browserFlow} />
+
+### After logout redirects
 
 After successful logout, the browser will be redirected either to the `return_to` query parameter from the initial request URL, or
-fall back to the `default_browser_return_url` value set in Ory Identities' configuration file:
+fall back to the default post logout return URL.
+
+```mdx-code-block
+<Tabs group="oc-vs-oss">
+  <TabItem value="oc" label="The Ory Network" default>
+```
+
+You can set a default return URL in the [Ory Console](https://console.ory.sh/projects/current/browser-redirects) on the **Browser
+Redirects** page using the **Post-Logout Redirect** value.
+
+```mdx-code-block
+  </TabItem>
+  <TabItem value="oss" label="Ory Identities">
```

Comment:
And this should show how to do it using the Ory CLI (**please test this yourself, i only wrote what i think should work**):

```
ory patch identity-config {project_id}
  --replace '/selfservice/flows/logout/after/default_browser_return_url="https://example.org"'
```

Reply:
Works. Done.